data/assets/scene/solarsystem/missions/newhorizons/actions.asset

```diff
@@ -0,0 +1,229 @@
+  ]],
+  Documentation = "Jumps to the 14th of July 2015 at 0900 UTC and clears all projections",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local increase_hightmap_pluto = {
+  Identifier = "os.missions.newhorizons.pluto.increasehightmap",
+  Name = "Pluto HeightExaggeration +",
+  Command = [[
+    openspace.setPropertyValueSingle(\"Scene.PlutoProjection.Renderable.HeightExaggeration\", openspace.getPropertyValue(\"Scene.PlutoProjection.Renderable.HeightExaggeration\") + 5000);
+  ]],
+  Documentation = "Increases the height map exaggeration on Pluto",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local decrease_hightmap_pluto = {
+  Identifier = "os.missions.newhorizons.pluto.decreasehightmap",
+  Name = "Pluto HeightExaggeration -",
+  Command = [[
+    openspace.setPropertyValueSingle(\"Scene.PlutoProjection.Renderable.HeightExaggeration\", openspace.getPropertyValue(\"Scene.PlutoProjection.Renderable.HeightExaggeration\") - 5000);
+  ]],
+  Documentation = "Decreases the height map exaggeration on Pluto",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local increase_hightmap_charon = {
+  Identifier = "os.missions.newhorizons.charon.increasehightmap",
+  Name = "Charon HeightExaggeration +",
+  Command = [[
+    openspace.setPropertyValueSingle(\"Scene.CharonProjection.Renderable.HeightExaggeration\", openspace.getPropertyValue(\"Scene.CharonProjection.Renderable.HeightExaggeration\") + 5000);
+  ]],
+  Documentation = "Increases the height map exaggeration on Charon",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local decrease_hightmap_charon = {
+  Identifier = "os.missions.newhorizons.charon.decreasehightmap",
+  Name = "Charon HeightExaggeration -",
+  Command = [[
+    openspace.setPropertyValueSingle(\"Scene.CharonProjection.Renderable.HeightExaggeration\", openspace.getPropertyValue(\"Scene.CharonProjection.Renderable.HeightExaggeration\") - 5000);
+  ]],
+  Documentation = "Decreases the height map exaggeration on Charon",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local toggle_pluto_trail = {
+  Identifier = "os.missions.newhorizons.pluto.toggletrail",
+  Name = "Toggle Pluto Trail",
+  Command = [[
+    openspace.setPropertyValueSingle('Scene.PlutoBarycentricTrail.Renderable.Enabled', not openspace.getPropertyValue('Scene.PlutoBarycentricTrail.Renderable.Enabled'));
+  ]],
+  Documentation = "Toggles the visibility of the trail behind Pluto",
+  GuiPath = "/Missions/New Horizons",
+  isLocal = false
+}
+
+local toggle_pluto_labels = {
+  Identifier = "os.missions.newhorizons.pluto.togglelabels",
+  Name = "Toggle Pluto Labels",
+  Command = [[
+    local list = {\"Scene.PlutoText.Renderable.Enabled\", \"Scene.CharonText.Renderable.Enabled\", \"Scene.HydraText.Renderable.Enabled\", \"Scene.NixText.Renderable.Enabled\", \"Scene.KerberosText.Renderable.Enabled\", \"Scene.StyxText.Renderable.Enabled\"}; 
```

Comment:
No need to escape the " since they appear in a [[ ]] string;   one of the nice things of Lua that they have multiple string markers that can be nested